docs/libs/gelly_guide.md

```diff
@@ -194,6 +194,27 @@ val edgeTuples = env.readCsvFile[String, String, Double]("path/to/edge/input")
 
 val graph = Graph.fromTupleDataSet(vertexTuples, edgeTuples, env)
 {% endhighlight %}
+
+* from a CSV file of Edge data and an optional CSV file of Vertex data. In this case, Gelly will convert each row from the Edge CSV file to an `Edge`, where the first field will be the source ID, the second field will be the target ID and the third field (if present) will be the edge value. Equivalently, each row from the optional Vertex CSV file will be converted to a `Vertex`, where the first field will be the vertex ID and the second field (if present) will be the vertex value.
+If the vertices have no associated value, set the `vertexValue` parameter to `false`. If the edges have no associated value, set the `edgeValue` parameter to `false`.
```

Comment:
The role of `vertexValues` in creating the vertex set from the edges is not discussed here.